7.1/Dockerfile

```diff
@@ -0,0 +1,110 @@
+# syntax = docker/dockerfile:experimental
+ARG SPRYKER_PHP_VERSION=7.1.27
+
+FROM php:${SPRYKER_PHP_VERSION}-fpm-alpine3.9
+
+ENV srcRoot /data
+
+RUN mkdir -p ${srcRoot}
+
+ENV PHP_RUN_DEPS \
```

Comment:
Use `ENV` for variables which need to be available during runtime of this container (or child containers) only. If variable is needed only during runtime, `ARG` is enough.
Please review all env variables and check if they are needed during runtime, if no - change them to build-time arguments instead.

Reply:
Done